### ISSUE TITLE
Adjusting the size of the outer SVG element

### ISSUE THREAD

**Issue by vonvlaho** [rism-ch/verovio#391]
Is there any other parameter to adjust the lines vertical alignment? Since we set the spacingStaff aren't on the same vertical position anymore, though merely noticeable. Thanks again for your support!
![screenshot](https://cloud.githubusercontent.com/assets/23097270/21384289/329d9ac6-c76a-11e6-9270-1b92443a7a2a.jpg)

```javascript
 var options = JSON.stringify({
                scale: 40,
                pageHeight: 5000,
                pageWidth: 2000,
                spacingStaff: 0,
                ignoreLayout: 0,
                border: 0,
                adjustPageHeight: 1
            })```

**Comment by lpugin** [comment]
Sorry, I cannot reproduce it. Would it be able to send a link to it, or the PAE and SVG output?

BTW spacingStaff should not do anything with one single staff in the system.

**Comment by vonvlaho** [comment]
This HTML should reproduce the issue properly. I recognized that if not setting the spacingStaff the white-space between the lines gets bigger, which produces a greater space between them and the sung text.
[example.html.zip](https://github.com/rism-ch/verovio/files/666286/example.html.zip)

**Comment by lpugin** [comment]
OK, the problem come from the size of the SVG outer element with reduced zoom and a rounding of its value. I will fix it.

**Comment by vonvlaho** [comment]
Great! Thanks for your fabulous work!

**Comment by lpugin** [comment]
Fixed with 408438320cf7d099ae242e369da6f7218faa95fc
